server/ERNI.PBA.Server.DataAccess/Model/Transaction.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace ERNI.PBA.Server.DataAccess.Model
+{
+    public class Transaction
+    {
+        public int Id { get; set; }
+
+        public int RequestId { get; set; }
+
+        public int BudgetId { get; set; }
+
+        public int UserId { get; set; }
```

Comment:
And here, does it make sense to have the userId, since we already know it from the budget?